frontend_egui/src/app.rs

```diff
@@ -175,6 +175,8 @@ impl SnowGui {
     const TOAST_DURATION: Duration = Duration::from_secs(3);
     const ZOOM_FACTORS: [f32; 8] = [0.5, 0.8, 1.0, 1.2, 1.5, 2.0, 3.0, 4.0];
     const SUBMENU_WIDTH: f32 = 175.0;
+    const SCALING_ALGORITHMS: [ScalingAlgorithm; 2] =
```

Comment:
If you derive `strum::EnumIter` on `ScalingAlgorithm` you can use `ScalingAlgorithm::iter()` so you won't need the array